### PR TITLE
ParseEmailFiles data_to_md fix

### PR DESCRIPTION
**Pull request by darkushin**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/26488

## Description
If the body text of the email was None, the script failed to run.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/CommonScripts/Scripts/ParseEmailFiles/ParseEmailFiles.py [hunk 1]
```diff
@@ -3276,14 +3276,14 @@ def data_to_md(email_data, email_file_name=None, parent_email_file=None, print_o
     md += u"* {0}:\t{1}\n".format('To', email_data.get('To') or "")
     md += u"* {0}:\t{1}\n".format('CC', email_data.get('CC') or "")
     md += u"* {0}:\t{1}\n".format('Subject', email_data.get('Subject') or "")
-    if 'Text' in email_data:
+    if 'Text' in email_data and email_data['Text']:
```
Comment (glicht) [inline Packs/CommonScripts/Scripts/ParseEmailFiles/ParseEmailFiles.py:3279]: Might be simpler and clearer to use: email_date.get('Text')

File: Packs/CommonScripts/Scripts/ParseEmailFiles/ParseEmailFiles.py [hunk 2]
```diff
@@ -3276,14 +3276,14 @@ def data_to_md(email_data, email_file_name=None, parent_email_file=None, print_o
...
         text = email_data['Text'].replace('<', '[').replace('>', ']')
         md += u"* {0}:\t{1}\n".format('Body/Text', text or "")
     if 'HTML' in email_data:
```
Comment (glicht) [inline Packs/CommonScripts/Scripts/ParseEmailFiles/ParseEmailFiles.py:3282]: For completeness probably worth doing the same check here.

File: Packs/CommonScripts/Scripts/ParseEmailFiles/ParseEmailFiles.py [hunk 3]
```diff
@@ -3589,7 +3589,8 @@ def handle_eml(file_path, b64=False, file_name=None, parse_only_headers=False, m
                 'From': extract_address_eml(eml, 'from'),
                 'Subject': convert_to_unicode(eml['Subject']),
                 'HTML': convert_to_unicode(html),
-                'Text': convert_to_unicode(text),
+                
+               : convert_to_unicode(text),
```
Comment (bakatzir) [inline Packs/CommonScripts/Scripts/ParseEmailFiles/ParseEmailFiles.py:3593]: isn't the key missing here?  or is it a glitch in the screen

File: Packs/CommonScripts/ReleaseNotes/1_2_6.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+#### Scripts
+##### ParseEmailFiles
+  - Fixed an issue where the body text of the email was None
```
Comment (bakatzir) [inline Packs/CommonScripts/ReleaseNotes/1_2_6.md:3]:
```suggestion
  - Fixed an issue where "None" values of the text body of the email were causing the script to fail.
```

**Comment by yuvalbenshalom** [comment]
merge with one unrelated test failure on 4.1.0: ZipFile-Test

**Pull request merged**